package.json

```diff
@@ -40,6 +40,7 @@
     "lodash": "^4.13.1",
     "node-resemble-js": "0.0.5",
     "nodeclient-spectre": "^1.0.3",
+    "phantomjs-prebuilt": "^2.1.16",
```

Comment:
Why was this move to a dependency?

Reply:
Hmm. I think this happened when I used yarn to add it. I will switch.